api/Worker.json

```diff
@@ -217,7 +217,7 @@
               "version_added": "60"
             },
             "edge": {
-              "version_added": null
+              "version_added": "18"
```

Comment:
Confirmed by `new Worker('script').onmessageerror` being null in Edge 18 and undefined in Edge 17.